nrf_wifi/fw_if/umac_if/inc/fmac_structs_common.h

```diff
@@ -105,6 +105,7 @@ struct nrf_wifi_fmac_otp_info {
 	unsigned int flags;
 };
 
+#define MAX_NUM_REG_CHANELS 42
```

Comment:
A comment that explains how did you arrive at this or when will it change would be good.

Reply:
Added comment